patches/net/minecraft/world/entity/animal/Parrot.java.patch

```diff
@@ -9,3 +26,25 @@
                      this.tame(p_29414_);
                      this.level().broadcastEntityEvent(this, (byte)7);
                  } else {
+@@ -342,7 +_,11 @@
+ 
+     public static SoundEvent getAmbient(Level p_218239_, RandomSource p_218240_) {
+         if (p_218239_.getDifficulty() != Difficulty.PEACEFUL && p_218240_.nextInt(1000) == 0) {
+-            List<EntityType<?>> list = Lists.newArrayList(MOB_SOUND_MAP.keySet());
++            final var entities = new java.util.HashSet<>(MOB_SOUND_MAP.keySet());
++            p_218239_.registryAccess().registryOrThrow(net.minecraft.core.registries.Registries.ENTITY_TYPE)
++                    .getDataMap(net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps.PARROT_IMITATIONS)
++                    .keySet().forEach(key -> entities.add(net.minecraft.core.registries.BuiltInRegistries.ENTITY_TYPE.get(key)));
```

Comment:
Nitpick: Is there any reason to use the registry from the level's registry access in one case and the builtin registry in the other case? Should probably be unified to use one or the other for both.